README.md

```diff
@@ -799,16 +503,60 @@ a:hover {
 /* Basic one-line comment */
 ```
 
-<a name="classname-conventions"></a>
-## Classname Conventions
 
-<a name="use-smacss-prefixes-to-distinguish-classes-with-different-roles"></a>
-### Use SMACSS prefixes to distinguish classes with different roles.
+<a name="object-oriented-css-oocss"></a>
+## Object-Oriented CSS (OOCSS)
+
+OOCSS advocates practices which are beneficial to flexibility, code reuse, maintenance, and performance. For more info, see the [OOCSS wiki](https://github.com/stubbornella/oocss/wiki).
+
+
+<a name="composition-with-multiple-classes"></a>
+### Composition with Multiple Classes
+
+One of the core ideas of Object-Oriented CSS is using multiple classnames on a single HTML element to synthesize new components from small, modular, focused styles. 
+
+Separate multiple classes with TWO spaces for readability:
+
+```html
+<!-- Good -->
+<div class="c-tutorial-nav  c-tutorial-nav--alt  u-pull-left">
+
+<!-- Bad - don't use single space -->
+<div class="c-tutorial-nav c-tutorial-nav--alt u-pull-left">
+
+<!-- Bad - not ordered by inheritance: "alt" extends the base. -->
+<div class="c-tutorial-nav--alt  c-tutorial-nav  u-pull-left">
+```
+
+The classes should follow this order:
+
+```html
+<div class="[block|element]  [modifier  [modifier]]  [state]  [utility]  [javascript hooks]  [test hooks]">
+```
+
+
+<a name="quick-summary-of-oocss"></a>
+### Quick Summary of OOCSS:
```

Comment:
Who is the intended audience of this? I don't really think we can expect music consumers to grok anything about oocss, fwiw.